### PR TITLE
Editorial: update AO parameter declaration notations

### PR DESCRIPTION
**Pull request by legendecas**
Removes the redundant steps of type assertions.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -121,11 +129,18 @@ <h1>[[Call]] ( _thisArgument_, _argumentsList_ )</h1>
 		</emu-note>
 	</emu-clause>
 
-	<emu-clause id="sec-wrappedfunctioncreate" aoid="WrappedFunctionCreate">
-		<h1>WrappedFunctionCreate ( _callerRealm_, _targetFunction_ )</h1>
-		<p>The abstract operation WrappedFunctionCreate takes arguments _callerRealm_ and _targetFunction_. It is used to specify the creation of new wrapped function exotic objects. It performs the following steps when called:</p>
+	<emu-clause id="sec-wrappedfunctioncreate" type="abstract operation">
```
Comment (leobalter) [inline spec.html:132]: is the aoid not needed here anymore?

**Review by leobalter — COMMENTED** [review]
LGTM with a quick question. Thanks for doing this work!

File: spec.html (hunk 1 above)
Comment (legendecas) [reply spec.html:132]: ecmarkup suggested that if the header is structured then the aoid is no longer needed.

Comment (legendecas) [reply spec.html]: Maybe I'm mistaken, I didn't find ecmarkup warning about this again :\. Reverted.

Comment (leobalter) [reply spec.html]: It's all fine! I honestly don't know the correct answer and appreciate your attention to details here.

**Pull request merged**